public/views/chat.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<body>
+<!-- Messages container -->
+      <div id="messages-card" class="mdl-card mdl-shadow--2dp mdl-cell mdl-cell--12-col mdl-cell--6-col-tablet mdl-cell--6-col-desktop">
+        <div id="chat_header" ng-click="resize()">Chat<span id="unread" class="mdl-badge" data-badge=""></span></div>
+        <div id="show_message" class="mdl-card__supporting-text mdl-color-text--grey-600">
+          <div id="messages">
+            <span id="message-filler"></span>
+          </div>
+          <form id="message-form">
+            <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+              <input class="mdl-textfield__input" type="text" id="message">
+              <label class="mdl-textfield__label" for="message">Message...</label>
```

Comment:
LGTM. You could howevere, have the "Message" label set as a placeholder instead.  It